src/plots/plots.js

```diff
@@ -1815,6 +1818,9 @@ plots.transition = function(gd, data, layout, traces, frameOpts, transitionOpts)
     }
 
     function interruptPreviousTransitions() {
+        // Fail-safe against purged plot:
```

Comment:
This could go elsewhere, but it's a trivially inexpensive one-liner, so I think it's a good idea.